src/Workload/Shared/Maestro.targets

```diff
@@ -49,12 +49,12 @@
     <MSBuild
         Targets="Restore"
         Projects="$(PkgMicrosoft_DotNet_Arcade_Sdk)\tools\SdkTasks\PublishBuildAssets.proj"
-        Properties="Configuration=$(Configuration);RepoRoot=$(MauiRootDirectory);VersionPrefix=$(_MauiDotNetVersion)"
+        Properties="Configuration=$(Configuration);RepoRoot=$(MauiRootDirectory);VersionPrefix=$(Version)"
```

Comment:
I _think_ we may need to set `DependsOnTargets="SetVersions"` on the `PushManifestToBuildAssetRegistry` target to make sure `$(Version)` has a value when this runs?